app/helpers/forms_helper.rb

```diff
@@ -34,7 +34,8 @@ def money_with_currency_field(form, money_method, options = {})
       default_currency: options[:default_currency] || "USD",
       disable_currency: options[:disable_currency] || false,
       hide_currency: options[:hide_currency] || false,
-      label:         options[:label] || "Amount"
+      label:         options[:label] || "Amount",
+      required:      options[:required] || false
```

Comment:
We'll need to update this improper usage for this to flow through correctly:

https://github.com/maybe-finance/maybe/blob/fd4011126422023228a9295b2d9b30bb1b9402cb/app/views/accounts/_form.html.erb#L8

See `required: "required"` should be -> `required: true`